app/views/reports/index.html.erb

```diff
@@ -417,4 +391,16 @@
         });
     });
 
+    $('#export-submission-report').on('click', function() {
+        var that = $(this);
+        var text = that.text();
+        var btn = that.button();
+        btn.button('disable');
+        btn.button({'label': 'Exporting... file will download momentarily'});
+        setTimeout(function() {
```

Comment:
Using a time-out seems reasonable for expediency, but it might be worth recording our hope to refactor.

```suggestion
        setTimeout(function() {
        // TODO: Investigate ways to detect start of download, and eliminate explicit wait
```

Reply:
Agreed, but given that we are responding to the request with a text file, there's no response to render that can update the DOM.  This is mainly just to prevent the admin from clicking the button multiple times and getting a whole bunch of files.